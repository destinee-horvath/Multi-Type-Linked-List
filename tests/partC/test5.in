NEW 2
fried
rice
NEW 3
chicken
noodle
soup
VIEW ALL
INSERT 2 -1 lettuce
INSERT 0 -2 {1}
NEW 4
partial
derivatives
{0}
2.71
VIEW-NESTED 2
VIEW-NESTED 0
VIEW ALL
REMOVE 0
REMOVE 1
REMOVE 2
VIEW 0
VIEW 1
REMOVE 0
VIEW 1
REMOVE 1