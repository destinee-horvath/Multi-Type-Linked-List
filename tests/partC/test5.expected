List 0: fried -> rice
List 1: chicken -> noodle -> soup
Number of lists: 2
List 0
List 1
INVALID COMMAND: INSERT
Nested 0: fried -> {List 1} -> rice
Nested 2: partial -> derivatives -> {List 0} -> 2.71
partial -> derivatives -> {fried -> {chicken -> noodle -> soup} -> rice} -> 2.71
fried -> {chicken -> noodle -> soup} -> rice
Number of lists: 3
Nested 0
List 1
Nested 2
INVALID COMMAND: REMOVE
INVALID COMMAND: REMOVE
List 2 has been removed.

Number of lists: 2
Nested 0
List 1
fried -> {List 1} -> rice
chicken -> noodle -> soup
List 0 has been removed.

Number of lists: 1
List 1
chicken -> noodle -> soup
List 1 has been removed.

Number of lists: 0
